frontend/src/enemies/Guy.js

```diff
@@ -0,0 +1,11 @@
+import Enemy from "./Enemy";
+
+class Guy extends Enemy {
+    constructor () {
+        super();
```

Comment:
I have to go back and review inheritance a little more, but I see how silly this is.

Reply:
Why silly?

Reply:
fixed by putting the arguments in super()